load-tests/route_guide_bidirectional_streaming/deployment/deployment-patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  name: no-name
```

Comment:
Don't we need a name here?.

Reply:
Can you check with them, I think we don’t need this.

Reply:
Apparently, the deployment-patch is used to indicate the `imagePullPolicy: Always` config. As they said, its now fixed in `c2c`. Will remove this.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/3a9a72758d48b27e0d247135dc32dcd471aebdf7